src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -2589,3 +2614,60 @@ static const char* const dhcp_state_table[_DHCP_STATE_MAX] = {
 };
 
 DEFINE_STRING_TABLE_LOOKUP_TO_STRING(dhcp_state, DHCPState);
+
+int sd_dhcp_client_set_lease_file(sd_dhcp_client *client, int dir_fd, const char *path) {
+        assert_return(client, -EINVAL);
+        assert_return(!path || (dir_fd >= 0 || dir_fd == AT_FDCWD), -EBADF);
+
+        if (!path) {
+                client->lease_dir_fd = safe_close(client->lease_dir_fd);
+                return 0;
+        }
+
+        if (!path_is_safe(path))
+                return -EINVAL;
+
+        _cleanup_close_ int fd = AT_FDCWD;
```

Comment:
Please add comment about why `AT_FDCWD` is set. See `sd_dhcp_server_set_lease_file()`.